src/testServer.js

```diff
@@ -0,0 +1,36 @@
+import Promise from 'bluebird'
+import child_process from 'child_process'
+import process from 'process'
+import path from 'path'
+
+/**
+ * Start a test server
+ */
+
+let _child, waiting
+
+export default function testServer(script="index.js", env={}) {
```

Comment:
It's just a style point, but I'd probably opt for a verb-object name here, to make the intent clearer (e.g. `startTestServer()`).